ApplicationConfig/Providers/SiteAccesses.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformUIBundle\ApplicationConfig\Providers;
+
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\MVC\Symfony\SiteAccess;
+use EzSystems\PlatformUIBundle\ApplicationConfig\Provider;
+
+class SiteAccesses implements Provider
```

Comment:
ok, makes sense to use first one if no languageCode is set on creation, however for picking content type name language the logic should use whole prioritized language list and fallback to main language.